phpcs.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
```

Comment:
If I remember well we can point `phpcs` into the correct configuration file without copy it out.
Why did you move it here?
The idea is use the original one downloaded by composer in order to be up to date with the original one